### DISCUSSION TITLE
Idea Statica plugin for Rhino 8 Grasshopper

### DISCUSSION THREAD

**Discussion by uptowndowntown** [idea-statica/ideastatica-public#418] [Rhino/Grasshopper] [answered]
Hello,

Having tested some of the functionality of the very good plugin, I notice that it currently doesn't run on Rhino 8. I understand the plugin is still a WIP, but are there any plans to support Rhino 8 and if so is there a preliminary timeframe? I'm using some new features related to the Grasshopper Python scripting component only available in Rhino 8 and therefore would be interested in having your plugin working in the newer environment.

Best regards

Mads

**Comment by nathanlukeIDEA** [comment]
Hi @uptowndowntown, Thanks for the positive feedback on the plugin. I will aim to create a release for Rhino 8 very soon. Hopefully, it will be available in the next couple of weeks.

**Reply by uptowndowntown** [reply]
Thank you, Nathan. Will give it a go later today. Much appreciated.

**Reply by uptowndowntown** [reply]
Hello Nathan,

I can confirm that the script runs on Rhino 8 with the adjustment you provided.
Thanks.

**Comment by nathanlukeIDEA** [comment, marked as answer]
Hi @uptowndowntown,

You can use the plugin with Rhino8 if you set the DotNetFramework to .Net Runtime.
https://www.rhino3d.com/en/docs/guides/netcore/#to-start-rhino-in-net-framework-once

![image](https://github.com/idea-statica/ideastatica-public/assets/93578548/3fb17cd7-4537-4303-ac44-d27435b1d85b)

We will try and make it compatibile with the latest development.

Cheers
Nathan
